hosting_docs/source/support/1-index.rst

```diff
@@ -0,0 +1,12 @@
+Support System
+==============
+
+Overview on CommCareHQ support system
+
+
+.. toctree::
+   :maxdepth: 3
+
+   /support/2-why-set-up-support-system
```

Comment:
All of the pages here could actually become a single page here. It's not too long.